tools/repo_parser/README.md

```diff
@@ -0,0 +1,49 @@
+# Repo Parser
+
+This will parse the contents of an OpenFeature repo and determine how they adhere to the spec. This can be gamed and assumes everyone is participating in good faith.
+
+We look for a `.specrc` file in the root of a repository to figure out how to find test cases that are annotated with the spec number and the text of the spec. We can then produce a report which says "you're covered" or details about how you're not covered. The goal of this is to use that resulting report to power a spec-compliance matrix for end users to vet SDK quality.
+
+## Usage
+
+```
+$ docker build -t specfinder .
```

Comment:
Do we want to publish a docker image to Github Packages [1] for this? 

[1] https://docs.github.com/en/actions/publishing-packages/publishing-docker-images

Reply:
I can help set this up if you're interested.

Reply:
Yes, I'd like to set this up, but I don't think it's part of this PR. I've filed #258